generators/server/templates/src/main/resources/config/application-dev.yml.ejs

```diff
@@ -193,12 +193,9 @@ spring:
     <%_ } _%>
     <%_ if (searchEngine === 'elasticsearch') { _%>
         elasticsearch:
-            cluster-name:
-            cluster-nodes: localhost:9300
             properties:
                 path:
-                    logs: <%= BUILD_DIR %>elasticsearch/log
-                    data: <%= BUILD_DIR %>elasticsearch/data
+                    home: <%= BUILD_DIR %>elasticsearch
```

Comment:
as this is a Spring Data Elasticsearch property, is this being used by Jest also?

Reply:
Yes. Spring Data Jest doesn’t reimplement anything in Spring Data Elasticsearch. It uses this property to create the embedded ES instance. If you don’t have spring.data.jest.uri defined (which we don’t in the dev profile), this property is required.